src/components/Card/Card.tsx

```diff
@@ -0,0 +1,30 @@
+import { Component } from 'react';
+import type { Character } from '../../AppTypes';
```

Comment:
```suggestion
import type { Character } from '@/AppTypes';
```
А здесь ожидал увидеть алиасы.

Reply:
алиасы будут добавлены в ветке hooks-and-routing